src/command_parser.ts

```diff
@@ -131,6 +131,9 @@ export class CommandParser {
 
   parse(args?: string[]): void {
     const validArgs = args ?? process.argv.slice(2);
+    if (validArgs.length === 0) {
```

Comment:
To display stderr and perform exit(1), I think it would be better to modify _internalParseAndValidate() instead of here.
```diff
     try {
+      if (args.length === 0) {
+        throw new ParseError("No command specified");
+      }
+
       const parsed = parseMultipleCommands({
```

Reply:
Yeah, you are right.